flink-staging/flink-hbase/pom.xml

```diff
@@ -112,6 +112,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
Is the HBase server dependency really required for any client that wants to write into HBase? This seems like a pretty bad design on the HBase side.

Can you tell us what fails when you omit this dependency?